packages/build/src/plugins/child/constants.js

```diff
@@ -39,7 +39,7 @@ const LOCAL_CACHE_DIR = '.netlify/cache/'
 
 const getFunctionsDist = function(baseDir) {
   if (isNetlifyCI()) {
-    return `${tmp()}/zisi-${DEPLOY_ID}`
```

Comment:
Does `tmp` and `tmpdir` resolve to same thing?

This was required for current remote CI functions to deploy correctly

Reply:
`os.tmp()` does not exist. It's a typo for `os.tmpdir()`.